docs/fa/examples.md

```diff
@@ -0,0 +1,97 @@
+# Examples
```

Comment:
@eliotwrobson One more thought: I think it would be good to rename this to `FA Examples` to distinguish between any additional examples pages we create in the future (e.g. for PDA or TM).

Reply:
Done here: https://github.com/caleb531/automata/pull/146/commits/2ce6816e437d8ca17b4e095ab4922e18eb4421f7